src/data/__tests__/calculate.test.ts

```diff
@@ -43,6 +47,7 @@ describe('calculate', () => {
     const scenario = 'Outdoor masked hangout with 2 people'
     const data: CalculatorData = {
       ...exampleLocation,
+      ...standardRiskBudget,
```

Comment:
it would be clearer to just put  `riskBudget: BUDGET_STANDARD` in each of these, rather than defining a single key object and then spreading it.

Even better, put riskBudget into exampleLocation (possibly rename exampleLocation to baseTestData) so you don't add overhead for new tests.

Reply:
Oh good. Yeah I like the baseData solution. That was my first thought but I didn't want to mess with what was already there. Thanks. Will do.